README.md

```diff
@@ -31,6 +31,37 @@ module.exports = {
 };
 ```
 
+### Utility Classes
+
+The simplest way to apply Garden Design Tokens is by using the
+[Tailwind utility classes](https://tailwindcss.com/docs/utility-first).
+
+```html
+<img
+  class="rounded-full h-8 w-8 border border-green-600"
+  src="avatar.png"
+  alt="Avatar image" />
```

Comment:
nit: Should the word "image" be removed to discourage redundant words read aloud on screen readers? I've seen the [img-redundant-alt](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/img-redundant-alt.md#img-redundant-alt) rule used in Garden and have come across that lint warning before.

Reply:
Good catch. Removed.